astroquery/esasky/core.py

```diff
@@ -130,6 +132,10 @@ def query_object_catalogs(self, position, catalogs=__ALL_STRING,
             Can be either a specific catalog or a list of catalogs (all catalog 
             names are found in list_catalogs()) or 'all' to search in all 
             catalogs. Defaults to 'all'. 
+        row_limit : int, optional
+            Determines how many rows that will be fetched from the database
+            for each mission. Can be -1 to select maximum (currently 100 000).
+            Defaults to 2000. 
```

Comment:
Maybe specify: "The server-set default, accessed by setting `row_limit='default'`, is 2000."  Is that correct?